trade-finance-manager-api/src/v1/controllers/amendment.controller.js

```diff
@@ -1,7 +1,7 @@
 const api = require('../api');
 const { createAmendmentTasks, updateAmendmentTasks } = require('../helpers/create-tasks-amendment.helper');
 const { isRiskAnalysisCompleted } = require('../helpers/tasks');
-const { amendmentEmailEligible, sendAutomaticAmendmentEmail, sendManualDecisionAmendmentEmail, sendManualBankDecisionEmail } = require('../helpers/amendment.helpers');
+const { amendmentEmailEligible, sendAutomaticAmendmentEmail, sendManualDecisionAmendmentEmail, sendManualBankDecisionEmail, sendFirstTaskEmail } = require('../helpers/amendment.helpers');
```

Comment:
I normally list functions if more than three in new lines, purely a personal preference not necessary.
Thought I'll add this as a suggestion.